internal/junitxml/report.go

```diff
@@ -17,8 +17,16 @@ import (
 
 // JUnitTestSuites is a collection of JUnit test suites.
 type JUnitTestSuites struct {
-	XMLName xml.Name `xml:"testsuites"`
-	Suites  []JUnitTestSuite
+	XMLName   xml.Name `xml:"testsuites"`
+	Name      string   `xml:"name,attr,omitempty"`
+	Timestamp string   `xml:"timestamp,attr"`
+	Tests     int      `xml:"tests,attr"`
+	Failures  int      `xml:"failures,attr"`
+	Errors    int      `xml:"errors,attr"`
+	Skipped   int      `xml:"skipped,attr"`
+	Time      float64  `xml:"time,attr"`
+	Hostname  string   `xml:"hostname,attr,omitempty"`
```

Comment:
I'm not sure this is correct. From my read of the XSD file, these should be properties of `JUnitTestSuite` not `JUnitTestSuites`. Am I missing something? Can you explain more about why they would be on `JUnitTestSuites` ?

`JUnitTestSuite` already has many of these fields, but we could add the ones that are missing.

Reply:
This is a little tricky. In some definitions are these properties available for `JUnitTestSuites` (e.g., https://github.com/junit-team/junit5/blob/main/platform-tests/src/test/resources/jenkins-junit.xsd) and for others not.
I am using this parser to generate a user-friendly view of the tests https://lotterfriends.github.io/online-junit-parser/ and there these properties are used for the title.

Reply:
Even in that jenkins schema, the `testsuites` only has name, time, tests, failures, disabled, and errors. It doesn't have hostname, timestamp, or skipped. Those are all properties of `testsuite`. 

Looking at the source of the tool you linked, 

https://github.com/lotterfriends/online-junit-parser/blob/d279015fac02c402ec12ce033f1a433508914b35/index.html#L122-L132

I don't see hostname in there either. Hostname is on the testsuite (not testsuites).  

So something still doesn't look right here.

I looked for title in that source, and I guess that's the `<h1>` here, is that right?

https://github.com/lotterfriends/online-junit-parser/blob/d279015fac02c402ec12ce033f1a433508914b35/index.html#L174

That seems to only use the name. Are you sure these other properties are required?